src/schema/image/index.js

```diff
@@ -25,6 +25,9 @@ const ImageType = new GraphQLObjectType({
   fields: () => ({
     aspect_ratio: {
       type: GraphQLFloat,
```

Comment:
```suggestion
      type: new GraphQLNonNull(GraphQLFloat),
```

As it will never return null now - but you'll need to import it at the top though